### ISSUE TITLE
Workstream definition: Base Profile specification

### ISSUE THREAD

**Issue by bettinaheim** [qir-alliance/.github#15]
This issue is created to capture any discussions or feedback also from the broader community regarding the creation of a workstream to specify a Base-Profile. The [workstream definition document](https://github.com/qir-alliance/.github/blob/main/workstreams/Base_Profile_Workstream.md) captures the goals, requirements, expected outcomes, as well as what is in and out of scope for the workstream. Please comment on this issue to share thoughts about the workstream itself (i.e. about the content of that document). A suitable issue on the spec repo to track the work for defining such a Base Profile will be created once the workstream goals are clear and the workstream is approved.

**Comment by bettinaheim** [comment]
Additional requirement: Document how to define additional quantum instruction sets.

**Comment by bettinaheim** [comment]
The workstream as defined in the document linked in the description has been approved by the steering committee. This issue is hence closed. If you are interested in the status and progress of the workstream, please see [this section](https://github.com/qir-alliance/.github/blob/main/workstreams/Base_Profile_Workstream.md#status--discussions) in the document.
